### PR TITLE
Deprecate a lot of global js

### PR DESCRIPTION
**Pull request by rullzer**
:musical_note: We don't need no water, let the .... :musical_note:

### CODE REVIEW

File: core/src/globals.js [hunk 1]
```diff
@@ -111,8 +111,8 @@ window['ClipboardJS'] = ClipboardJS
 window['dav'] = dav
 setDeprecatedProp('DOMPurify', () => DOMPurify, 'The global DOMPurify is deprecated, ship your own')
 window['Handlebars'] = Handlebars
-window['jstimezonedetect'] = jstimezonedetect
-window['jstz'] = jstimezonedetect
+window['jstimezonedetect'] = deprecate(jstimezonedetect, 'jstimezonedetect')
```
Comment (ChristophWurst) [inline core/src/globals.js]: :warning: please change this helper to take a version as parameter. It's hard coded to 19

Comment (rullzer) [reply core/src/globals.js]: Done. Also deprecated more.

**Review by skjnldsv — APPROVED** [review]
:heart:

**Comment by ChristophWurst** [comment]
Mind using `setDeprecatedProp` instead? Then you get the warning as soon as you access the global. Reasoning: not everything is a function. Prime example: Backbone. You might use `Backbone.Model` or `Backbone.Collection`, but never do a `Backbone()`, hence the `deprecate` helper won't catch the access.

**Comment by rullzer** [comment]
> Mind using `setDeprecatedProp` instead? Then you get the warning as soon as you access the global. Reasoning: not everything is a function. Prime example: Backbone. You might use `Backbone.Model` or `Backbone.Collection`, but never do a `Backbone()`, hence the `deprecate` helper won't catch the access.

right makes sense.

**Review by skjnldsv — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**